src/main/PDF/Services/AnnotationServices/UploadSignedPDFService.java

```diff
@@ -88,7 +87,7 @@ public Message executeAndGetResponse() {
               || privilegeLevel == UserType.Director
               || privilegeLevel == UserType.Admin)) {
         try {
-          InputStream signedPDF = signPDF(uploader, fileStream, signatureFileStream);
+          InputStream signedPDF = signPDF(clientUsername, fileStream, signatureFileStream);
```

Comment:
I notice here that you are putting in the client Username into the signPDF function, can you assert in the integration test that the signature field is whatever client username you are passing into request body? On this line: https://github.com/keepid/keepid_server/blob/781334223b09257896877174b284528295575844/src/main/PDF/Services/AnnotationServices/UploadSignedPDFService.java#L156 I see we set some field on the signature here, so if you could assert this is correctly changed that would be awesome